pkg/encoder/base64.go

```diff
@@ -6,16 +6,18 @@ import "encoding/base64"
 // base64 encoding strategy.
 type Base64Encoder struct{}
 
+var _ Encoder = (*Base64Encoder)(nil)
+
 // NewBase64Encoder constructs an Encoder that implements a base64 encoding as specified
 // by the encoding/base64 package.
 func NewBase64Encoder() *Base64Encoder {
 	return &Base64Encoder{}
 }
 
-func (b *Base64Encoder) Encode(val []byte) (string, error) {
-	return base64.URLEncoding.EncodeToString(val), nil
+func (e *Base64Encoder) Decode(s string) ([]byte, error) {
```

Comment:
Can we add some godoc here such as? I'm going to start motivating us to add public godoc for anything not in the `internal` package.

```
// Decode base64 URL decodes the provided string.
```

Reply:
Good point. Added! Thanks.